svcadm

```diff
@@ -1,27 +1,36 @@
-# svcadm completion                                        -*- shell-script -*-
-#
+# OpenIndiana svcadm(1M) & svcs(1) completions              -*- shell-script -*-
+# ------------------------------------------------------------------------------
 # Copyright 2006 Yann Rouillard <yann@opencsw.org>
 # Portions copyright 2013, Nexenta Systems, Inc.
+# Copyright (c) 2018, Michal Nowak <mnowak@startmail.com>
 
-# 
-# svcadm accept any complete FMRI or abreviated FMRI
-#  - a complete FMRI is svc:/foo/bar/bar/baz
-#  - abbreviated FMRI are foo/bar/bar/baz, bar/bar/baz, bar/baz or baz
+# svcadm accepts any complete FMRI or abreviated FMRI
+#   - a complete FMRI is svc:/foo/bar/bar/baz,
+#   - abbreviated FMRI are foo/bar/bar/baz, bar/bar/baz, bar/baz or baz.
 #
-# The goal of this function is to be able to propose all alternatives,
+# The goal of this function is to propose all alternatives,
 # but to not clutter the interface with all completions, we will only
 # cut every completion alternative at the next slash.
 #
-# For exemple, if the user types <nothing><tab>, we will propose for svc://foo/bar/bar/baz
-# the following completion: foo/, bar/ and baz
-# If the user types <b><tab>, we will propose: bar/ and baz
-# If the user types <bar/><tab>, we will propose: bar/bar/ and bar/baz
-#
-# By default, the function proproses only abbreviated completions except if the user already
-# began to type svc:. In that case we will propose only the complete FMRI beginning with the
-# pattern
+# For exemple, if the user types <nothing><tab>, for the svc://foo/bar/bar/baz
```

Comment:
Looks like a French speaking person wrote the text :) Coud you just fix "exemple" -> "example"

Reply:
Thanks :), updated couple more typos.